### PR TITLE
✨Rename field volume_ml to volume_ul

### PR DESCRIPTION
**Pull request by parimalak**

### CODE REVIEW

**Review by znatty22 — APPROVED** [review]
💯

**Review by dankolbman — CHANGES_REQUESTED** [review]
Needs a migration.

File: migrations/versions/cdbd609e2bd5_.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('biospecimen', sa.Column(
+        'volume_ul', sa.Float(), nullable=True))
+    op.drop_column('biospecimen', 'volume_ml')
+    op.add_column('biospecimen_diagnosis', sa.Column(
```
Comment (znatty22) [inline migrations/versions/cdbd609e2bd5_.py]: This looks like it maybe shouldn't be here? I thought you made that change to the model in another PR we already merged

Comment (dankolbman) [inline migrations/versions/cdbd609e2bd5_.py]: why is this here?

File: migrations/versions/cdbd609e2bd5_.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Revision ID: cdbd609e2bd5
+Revises: ce01ada6ac80
+Create Date: 2018-08-27 10:28:59.687358
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+import dataservice
```
Comment (znatty22) [inline migrations/versions/cdbd609e2bd5_.py:12]: Doesn't seem to be used anywhere

File: migrations/versions/cdbd609e2bd5_.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+down_revision = 'ce01ada6ac80'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('biospecimen', sa.Column(
+        'volume_ul', sa.Float(), nullable=True))
+    op.drop_column('biospecimen', 'volume_ml')
```
Comment (dankolbman) [inline migrations/versions/cdbd609e2bd5_.py]: We want to rename, not drop, or else we will lose all the data.

**Review by dankolbman — CHANGES_REQUESTED** [review]
I think you want to use the rename column function instead of dropping and adding a new column.

File: migrations/versions/cdbd609e2bd5_.py (hunk 3 above)
Comment (parimalak) [reply migrations/versions/cdbd609e2bd5_.py]: good catch, apparently alembic doesn't automatically recognize the column name changes will change this manually.

File: migrations/versions/cdbd609e2bd5_.py (hunk 1 above)
Comment (parimalak) [reply migrations/versions/cdbd609e2bd5_.py]: I think this was missed in previous migration

File: migrations/versions/cdbd609e2bd5_.py (hunk 2 above)
Comment (parimalak) [reply migrations/versions/cdbd609e2bd5_.py:12]: use this import while upgrading database

**Review by znatty22 — APPROVED** [review]
👍

**Review by dankolbman — APPROVED** [review]
lgtm

**Comment by baileyckelly** [comment]
@rosibaj This change is in QA now (Biospecimen -> volume_ul)
